aries-site/src/pages/templates/drill-down-navigation.mdx

```diff
@@ -1,3 +1,50 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
+import { DrillDownCardExample, DrillDownDataTableExample, DrillDownListExample } from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+- When users need to focus on one task at a time.
+- To give information overview with the ability to drill down into specific information.
+- When screen real estate is limited.
+- When designing for many levels of functionality.
+- To divide meaningful content into sections.
+
+## Guidelines:
+- When the user is moving forward step-by-step to access further information.
+- When going from your Account > Dashboard > Service.
+- To allow users to dive deeper into related content.
+- Can be used as an individual element or in groups of related content.
+- Each element should have it’s own single content.
+
+## Drill Down Navigation Examples:
+
+### Card Example
+A visual representation of interactive and organized content.
+
+<Example caption="A diagram illustrating drill down navigation within Cards." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
+<DiagramPreview src={{dark: "/templateImages/drill-down-card-example.svg", light: '/templateImages/drill-down-card-example-invert.svg'}} alt="A diagram illustrating drill down navigation within cards." />
+</Example>
+
+<Example plain>
+<DrillDownCardExample />
+</Example>
+
+### List Example
+Lists provide access points that present a collection of organized and related content.
+<Example caption="A diagram illustrating drill down navigation within Lists" plain pad="small" previewWidth="large" height={{ min: 'small' }}>
+<DiagramPreview src={{dark: "/templateImages/drill-down-list-example.svg", light: '/templateImages/drill-down-list-example-invert.svg'}} alt="A diagram illustrating drill down navigation within lists." />
+</Example>
+
+<Example plain>
+<DrillDownListExample />
+</Example>
+
+### DataTable Example
+Datatables present data in a column and row format. A user can be taken to a spoke when an entire table row is clickable or if an individual cell is clickable to drill down into more information.
+
+<Example caption="A diagram illustrating drill down navigation within a DataTable." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
```

Comment:
There seems to be a problem with the image not showing. I'm on Chrome.
<img width="922" alt="Screen Shot 2022-01-11 at 2 12 20 PM" src="https://user-images.githubusercontent.com/62117454/149029995-750065d1-cfa6-4ca8-8143-e6893d8f81ad.png">

Reply:
Firefox:
<img width="1007" alt="Screen Shot 2022-01-11 at 2 16 40 PM" src="https://user-images.githubusercontent.com/62117454/149030212-a5a0c5c2-76cd-4e9b-ac40-299241c0a174.png">


Safari:
<img width="949" alt="Screen Shot 2022-01-11 at 2 16 59 PM" src="https://user-images.githubusercontent.com/62117454/149030187-0f128a6f-16b3-4a96-a558-de2fcc7bbb16.png">

Reply:
Just had a typo on the mdx file when I called the image